manifests/resource/mailhost.pp

```diff
@@ -122,22 +120,10 @@
   String $proxy_pass_error_message               = 'off',
   Array $server_name                             = [$name]
 ) {
-
   if ! defined(Class['nginx']) {
     fail('You must include the nginx base class before using any defined resources')
   }
 
-  $root_group = $nginx::root_group
-
-  File {
```

Comment:
There are no `File` resources in this manifest (and resource defaults like this are discouraged anyway)